servers/visual_server.cpp

```diff
@@ -2411,6 +2411,18 @@ VisualServer::VisualServer() {
 	GLOBAL_DEF("rendering/quality/depth_prepass/disable_for_vendors", "PowerVR,Mali,Adreno,Apple");
 
 	GLOBAL_DEF("rendering/quality/filters/use_nearest_mipmap_filter", false);
+
+	GLOBAL_DEF("rendering/batching/gles2/use_batching", true);
```

Comment:
Is actually any merit in making it optional? Should not the code be simpler if it only worked with batching?

Reply:
It's configurable for debugging purposes only. I think the goal is to remove the old code path once we got enough testing.

Reply:
You are absolutely right but for testing at first we wanted to make it optional so we could fix any regressions first.